src/Cake.Hosts.Tests/CakeHostsTests.cs

```diff
@@ -56,6 +56,32 @@ public void RecordExists_CommentedHost_False()
             result.Should().BeFalse();
         }
 
+        [Fact]
+        public void GetAllRecords_Find_Records()
```

Comment:
There should be an assertive test, confirming that returned values indeed are contained in the file. 
One of the lines in the file should be an invalid record, another line should be a comment - those should be ignored
And at least 2 records in file and check that those are returned in the list